src/Home.js

```diff
@@ -59,7 +59,7 @@ class Home extends Component {
       const newAcronyms = R.set(likesLens, R.append(userId, likesView), acronyms);
 
       this.setState({
-        newAcronyms
+        acronyms: newAcronyms
```

Comment:
Which bug does this fix?

Reply:
been awhile but somewhere along the lines I think I accidentally renamed it or something because `this.state.acronyms` is what is read, not `this.state.newAcronyms`